File: go/domestic/01_verify_bank_account.go
package main

import (
	"encoding/json"
	"fmt"
)

func main() {
	baseURL := "https://sandbox.easylink.id"
	appKey := "YOUR_APP_KEY"
	privateKeyPem := "/path/to/private.pem"
	accessToken := "YOUR_ACCESS_TOKEN"

	payload := map[string]interface{}{
		"bankCode":      "BCA",
		"accountNumber": "1234567890",
	}

	fmt.Println("Verifying Bank Account...")
	_, res, err := SendEasylinkRequest(baseURL, "/v2/transfer/verify-bank-account", "POST", payload, appKey, privateKeyPem, accessToken)
	if err != nil {
		fmt.Printf("Error: %v\n", err)
	} else {
		resBytes, _ := json.MarshalIndent(res, "", "  ")
		fmt.Printf("Response:\n%s\n", string(resBytes))
	}
}

File: php/domestic/01_verify_bank_account.php
<?php

declare(strict_types=1);

require_once __DIR__ . '/../helpers.php';

// --- Configuration ---
$baseUrl       = 'https://sandbox.easylink.id';
$appId         = 'lQNJ0nL07Ucmemaa';
$appSecret     = 'HrfFeuRmoyBsZhxDi3w3JNdxwYu19lL4';
$appKey        = '3f9a7f74-de23-4fde-af75-da7684528a59';
$privateKeyPem = '/path/to/private.pem';

// Auto-fetch Access Token (or assign string token manually: $accessToken = 'YOUR_ACCESS_TOKEN';)
$accessToken   = getAccessToken($baseUrl, $appId, $appSecret);

$payload = [
    'bankCode'      => 'BCA',
    'accountNumber' => '1234567890'
];

echo "Verifying Bank Account...\n";
try {
    $res = sendEasylinkRequest(
        $baseUrl,
        '/v2/transfer/verify-bank-account',
        'POST',
        $payload,
        $appKey,
        $privateKeyPem,
        $accessToken
    );
    echo "Response:\n" . json_encode($res['data'], JSON_PRETTY_PRINT) . "\n";
} catch (Exception $e) {
    echo "Error: " . $e->getMessage() . "\n";
}
